MediaBrowser.Model/Entities/ProviderIdsExtensions.cs

```diff
@@ -9,6 +10,15 @@ namespace MediaBrowser.Model.Entities
     /// </summary>
     public static class ProviderIdsExtensions
     {
+        /// <summary>
+        /// Case insensitive dictionary of <see cref="MetadataProvider"/> string representation.
+        /// </summary>
+        private static Dictionary<string, string> metadataProviderEnumDictionary = new Dictionary<string, string>(
+            Enum.GetValues(typeof(MetadataProvider))
+                .Cast<MetadataProvider>()
+                .ToDictionary(enumValue => enumValue.ToString(), enumValue => enumValue.ToString()),
+            StringComparer.OrdinalIgnoreCase);
```

Comment:
```suggestion
        private static readonly Dictionary<string, string> _metadataProviderEnumDictionary =
            Enum.GetValues<MetadataProvider>()
                .ToDictionary(
                    enumValue => enumValue.ToString(),
                    enumValue => enumValue.ToString(),
                    StringComparer.OrdinalIgnoreCase);
```


also note the change from `metadataProviderEnumDictionary` -> `_metadataProviderEnumDictionary`